shuttle/templates/shuttle_stats.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'shuttle_base.html' %}
+
+{% block body_content %}
+    <div class="row">
+        <div class="col">
+            <div class="card">
+                <div class="card-body">
+                    <h4 class="card-title">Shuttle Check In</h4><hr>
+                    <p class="card-text">The Shuttle was last checked in at: </p>
+                    <h5>*Insert location here*</h5><hr>
+                    <p class="card-text">At the time of: </p>
+                    <h5>*Insert time here*</h5>
+                </div>
+            </div>
+        </div>
+        <div class="col">
+            <div class="card">
+                <div class="card-body">
+                    <h4 class="card-title">Shuttle Current Route</h4><hr>
+                    <p class="card-text">The Shuttle is in route to: </p>
+                    <h5>*Insert location here*</h5><hr>
+                    <p class="card-text">And should arrive at: </p>
+                    <h5>*Insert time here*</h5>
+                </div>
+            </div>
+        </div>
+        <div class="col">
+            <div class="card" >
+                <div class="card-body">
+                    <h4 class="card-title">Shuttle Schedule</h4><hr>
+                    <h5>*Insert schedule here*</h5>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock %}
```

Comment:
Could you move your html file inside a new directory inside of templates called `schedules`. So the path would be /templates/schedules/shuttle_stats.html.